app/_src/kubernetes-ingress-controller/deployment/minikube.md

```diff
@@ -30,14 +30,21 @@ Next, we will setup an environment variable with the IP address at which
 Kong is accessible. This will be used to actually send requests into the
 Kubernetes cluster.
 
+1. Get the IP address at which Kong is accessible:
 ```bash
-$ export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)
+$ minikube service -n kong kong-proxy --url | head -1
 # If installed by helm, service name would be "<release-name>-kong-proxy".
-# $ export PROXY_IP=$(minikube service <release-name>-kong-proxy --url | head -1)
-$ echo $PROXY_IP
+# minikube service <release-name>-kong-proxy --url | head -1
+```
```

Comment:
Can you please indent the codeblocks in both steps by three spaces?

Reply:
Thank you for the review @lena-larionova . Just committed the changes.